allure-report-face/src/test/webapp/core/testcase/testcaseSpec.js

```diff
@@ -213,6 +213,33 @@ describe('Testcase controllers', function() {
             expect(scope.testcase.steps[1].steps[1].failure.error).toBe(scope.failure.error);
         });
 
+        describe('hasParametersOfKind checks', function () {
+            var testcase = {
+                parameters: [
+                    {kind: 'argument'},
+                    {kind: 'system-property'}
+                ]
+            };
+
+            beforeEach(function() {
+                scope = createController({steps: []});
+            });
+            it('should return [true]', function () {
+                expect(scope.hasParametersOfKind(testcase, 'argument')).toBe(true);
+            });
+            it('should return [false], item not found', function () {
+                expect(scope.hasParametersOfKind(testcase, 'environment-variable')).toBe(false);
+            });
+            it('should return [false], no items in array', function () {
+                testcase.parameters = [];
+                expect(scope.hasParametersOfKind(testcase, 'argument')).toBe(false);
+            });
+            it('should return [false], parameters are undefined', function () {
+                var testcase = {};
+                expect(scope.hasParametersOfKind(testcase, 'argument')).toBe(false);
+            });
+        });
+
```

Comment:
You've removed `hasParametersOfKind`. This tests would fail.

Now here just enough one test which checks, that it properly fetch argument parameter from others